docs/content/any/guides/more/testing/index.md

```diff
@@ -0,0 +1,163 @@
+---
+date: '2021-12-04T01:58:28.010Z'
+docname: using/examples/testing
+images: {}
+path: /using-examples-testing
+title: Test Your Policy
+weight: 10
+description: |
+  Learn to test your Oso policies.
+aliases:
```

Comment:
I don't think you need this alias. We just add these when there's been a previous page that needs a redirect.